masakari-controller/controller/masakari_util.py

```diff
@@ -663,52 +702,54 @@ def fetch_servers_on_hypervisor(self, hypervisor):
             'all_tenants': True,
             }
         try:
-            self.rc_util.syslogout('Fetch Server list on %s' % hypervisor,
-                                   syslog.LOG_INFO)
+            msg = ('Fetch Server list on %s' % hypervisor)
+            LOG.info(self.rc_util.msg_with_thread_id(msg))
             servers = self.nova_client.servers.list(detailed=False,
                                                     search_opts=opts)
+
             return [s.id for s in servers]
 
         except exceptions.ClientException as e:
-            error_code = "[RecoveryControllerUtilApi_0005]"
             msg = 'Fails to call Nova Servers List API: %s' % e
-            self.rc_util.syslogout(error_code + msg, syslog.LOG_ERR)
+            LOG.error(self.rc_util.msg_with_thread_id(msg))
             raise
 
+    @log_process_begin_and_end.output_log
     def disable_host_status(self, hostname):
         """Disable host' status.
 
         :hostname: Target host name
         """
         try:
-            self.rc_util.syslogout('Disable nova-compute on %s' % hostname,
-                                   syslog.LOG_INFO)
+            msg = ('Disable nova-compute on %s' % hostname)
+            LOG.info(self.rc_util.msg_with_thread_id(msg))
             self.nova_client.services.disable(hostname, 'nova-compute')
+
         except exceptions.ClientException as e:
-            error_code = "[RecoveryControllerUtilApi_0006]"
             msg = 'Fails to disable nova-compute on %s: %s' % (hostname, e)
-            self.rc_util.syslogout(error_code + msg, syslog.LOG_ERR)
+            LOG.error(self.rc_util.msg_with_thread_id(msg))
             raise
 
+    @log_process_begin_and_end.output_log
     def do_instance_evacuate(self, uuid, targethost):
         """Call evacuate API for server instance.
 
         :uuid : Instance id
         :targethost: The name or ID of the host where the server is evacuated.
         """
         try:
-            self.rc_util.syslogout('Call Evacuate API with %s to %s' %
-                                   (uuid, targethost), syslog.LOG_INFO)
+            msg = ('Call Evacuate API with %s to %s' %
+                                   (uuid, targethost))
+            LOG.info(self.rc_util.msg_with_thread_id(msg))
             self.nova_client.servers.evacuate(uuid, host=targethost,
                                               on_shared_storage=True)
+
         except exceptions.ClientException as e:
-            error_code = "[RecoveryControllerUtilApi_0007]"
             msg = ('Fails to call Instance Evacuate API onto %s: %s'
                    % (targethost, e))
-            self.rc_util.syslogout(error_code + msg, syslog.LOG_ERR)
+            LOG.error(self.rc_util.msg_with_thread_id(msg))
             raise
 
-
 class RecoveryControllerUtil(object):
```

Comment:
Just a question. Is there any reason the class is not a sub class of Logger of oslo.logging?  I think we don't need to write long code, not LOG.error(self.rc_util.msg_with_thread_id()) but just LOG.error() , if this instance is Logger class and is instantiated as LOG.